app/src/pages/inside/testsPage/testsPage.jsx

```diff
@@ -122,7 +141,7 @@ export class TestsPage extends Component {
     return (
       <PageLayout>
         <PageSection>
-          <HistoryLine />
+          <HistoryLine onItemSelect={this.props.changeActiveLogItemAction} />
```

Comment:
Why is it still on tests page?
If storybook is not enough, you can move it to the sandbox page.

Reply:
Fixed